### PR TITLE
Feat: Create About.jsx

### LINKED ISSUE #96: Feat: View About page
## Description
As a public user,
I need see `About` page,
so that I can see infomation about the BridgeInTech Application.

## Mocks
Based on [Target MVP](https://www.figma.com/file/jxCqu1lq55H7jC4FDY119a/BridgeInTech?node-id=10%3A1), add a content describing what BridgeInTech application is about: 
Screen Shot 2020-09-16 at 4 35 57 pm

### Update [Required]
- [ ] Add content to the `About` page (sub menu under Home).
- [ ] Although the design is not the main focus, a reasonable layout is expected. 
- [ ] Avoid changing the Navbar layout in this PR as it is not required
- [ ] Link to `Read more about the project here...` with the [BridgeInTech Frontend Wiki repository](https://github.com/anitab-org/bridge-in-tech-web/wiki).

### Enhancement to Update [Optional]
- [ ] Image can be taken from the source refered on the Figma MVP above or can be replaced with other imagee (IMPORTANT: Please be careful with copyright issue. Make sure you give a visual refence to the source of image).

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
1 day

**Additional Information**
To do this task, you do not need to setup BridgeInTech Backend environment (if you haven't already done so), since there is no login required.

### PR DESCRIPTION
**Pull request by HeroicHitesh**
### Description

- Create `About.jsx` for About section in `home` folder
- Add  `/about` route in `Routes.js`
- Move `Home.jsx` to `home` folder
- update `react-scripts` dependency from version `3.4.1` to `^3.4.3` in `package.json`

Fixes #96 

### Type of Change:

- Code
- User Interface

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

### Screenshot of change:

Screenshot 2020-09-17 at 12 50 38 PM

### CODE REVIEW

**Review by mtreacy002 — CHANGES_REQUESTED** [review]
@HeroicHitesh , I can't see the image being displayed when I run the app. It looks like you need to add the image file locally inside the `home` folder and refer to it as the `src` address.

Actually..... it's now visible (after waiting for a while). I wonder if it's better to add image to folder or use online refence 🤔

Perhaps we can keep it as it is for now, and only change it later when it becomes an issue.

Other than that, this PR looks good 👍  and I noticed that you've made the navbar looks `prettier`, which is awesome ✨✨. 
But I'd like to ask you to change one thing. It's not a fault from your side, I just have a change of mind with the url link to `Read more about the projeect here...`. Initially I put the BridgeInTech repo as link since I'm thinking as a developer. But our user might not be interested on contributing to BridgeInTech project as developer, instead they want to use BridgeInTech as a user (organization/mentor/mentee). In this situation, an intro to what the application is about and benefit seems to be more useful for our user. 

For that purpose, can you please change the link to `https://medium.com/anitab-org-open-source/bridgeintech-by-anitab-org-85745853865d?source=friends_link&sk=fcb616ef0ac7e1e85ac6b58b513a56ee` instead? This is my medium blog post for [Introduction to BridgeInTech](https://medium.com/anitab-org-open-source/bridgeintech-by-anitab-org-85745853865d?source=friends_link&sk=fcb616ef0ac7e1e85ac6b58b513a56ee) from an application overview.

I really appreciate your help. 👍

**Comment by HeroicHitesh** [comment]
> @HeroicHitesh , I can't see the image being displayed when I run the app. It looks like you need to add the image file locally inside the home folder and refer to it as the src address.
>
> Actually..... it's now visible (after waiting for a while). I wonder if it's better to add image to folder or use online refence 🤔
>
> Perhaps we can keep it as it is for now, and only change it later when it becomes an issue.

@mtreacy002 I read this in one of the GitHub repo's PR reviews that it's better to use online reference instead of statically adding content, to reduce the total repository size. 
Also, I have one more suggestion if we are using AWS we can store static content in S3, which will not increase repository size and I hope there will be no display delay as it Amazon😅

> For that purpose, can you please change the link to `https://medium.com/anitab-org-open-source/bridgeintech-by-anitab-org-85745853865d?source=friends_link&sk=fcb616ef0ac7e1e85ac6b58b513a56ee` instead? This is my medium blog post for Introduction to BridgeInTech from an application overview.

Done👍

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "3.4.1"
+    "react-scripts": "^3.4.3"
```
Comment (mtreacy002) [inline package.json:15]: any reason why we need to update the version here? just curious.

File: src/home/About.jsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+          mentorship program to a mentor and a mentee that is customised to meet
+          the needed skills set within its organisation while providing a safety
+          and supportive environment for these mentor/mentee to work in.
+          Although the main focus is to empower women in tech, following
+          AnitaB.org's values that welcomed all allies who support equity in
+          tech, this application is also open to anyone that consider themselves
+          allies.
+        </p>
+        <a
+          href="https://medium.com/anitab-org-open-source/bridgeintech-by-anitab-org-85745853865d?source=friends_link&sk=fcb616ef0ac7e1e85ac6b58b513a56ee"
```
Comment (mtreacy002) [inline src/home/About.jsx:43]: Thank you for accommodating with the request, @HeroicHitesh 👍

**Review by mtreacy002 — COMMENTED** [review]
Hi @HeroicHitesh, just need one more confirmation from you.

File: package.json (hunk 1 above)
Comment (HeroicHitesh) [reply package.json:15]: I ran `npm install` and it showed some warnings so I ran `npm audit fix` and that updated this version which I guess will reduce any vulnerabilities we may find in the future. Also, adding `^` helps use any version greater than or equal to the mentioned one

**Comment by mtreacy002** [comment]
> @mtreacy002 I read this in one of the GitHub repo's PR reviews that it's better to use online reference instead of statically adding content, to reduce the total repository size.

Makes sense...

> Also, I have one more suggestion if we are using AWS we can store static content in S3, which will not increase repository size and I hope there will be no display delay as it Amazon😅

That's also my wish, @HeroicHitesh, for this purpose and also for the purpose of having reliable remote servers .... cc @anitab-org/bridgeintech-maintainers and @anitab-org/bridgeintech-managers 🙏

File: package.json (hunk 1 above)
Comment (mtreacy002) [reply package.json:15]: I see. In that case, can you please add to the PR description under `Type of Change` that extra info: "update react-scripts dependency from version 3.4.1 to 3.4.3" follows by instruction for other contributors to update this dependency locally

Comment (HeroicHitesh) [reply package.json:15]: Done

**Review by mtreacy002 — APPROVED** [review]
You did a great job, @HeroicHitesh on your First-Timers-Only issue. I'm really happy that you''re contributing to the project 🎉. Look forward to your next contribution 😉

**Comment by HeroicHitesh** [comment]
> You did a great job, @HeroicHitesh on your First-Timers-Only issue. I'm really happy that you''re contributing to the project 🎉. Look forward to your next contribution 😉

Thank you for the kind words, looking forward to contributing to more :electron: `React`ive issues to improve the platform

**Comment by mtreacy002** [comment]
> > You did a great job, @HeroicHitesh on your First-Timers-Only issue. I'm really happy that you''re contributing to the project 🎉. Look forward to your next contribution 😉
> 
> Thank you for the kind words, looking forward to contributing to more :electron: `React`ive issues to improve the platform

Don't forget there're many ways you could contribute to the project, as mentioned on [zulip post here](https://anitab-org.zulipchat.com/#narrow/stream/223071-newcomers/topic/Want.20to.20contribute.3F/near/210336297). Perhaps start a discussion about the design on the #bridge-in-tech stream on the Navbar improvement you mentioned before? 😁

**Comment by HeroicHitesh** [comment]
> Don't forget there're many ways you could contribute to the project, as mentioned on zulip post here. 

True. Actually I am more inclined towards resolving all the React issues, so I mentioned that apart from that I'll surely contribute wherever I can.

> Perhaps start a discussion about the design on the #bridge-in-tech stream on the Navbar improvement you mentioned before? 😁

I guess you confused me with `Prathmesh Khandelwal` he started Navbar discussion [here](https://anitab-org.zulipchat.com/#narrow/stream/237630-bridge-in-tech/topic/New.20Issues/near/210322643)
Or are you talking about restructuring the project components which I mentioned [here](https://github.com/anitab-org/bridge-in-tech-web/issues/96#issuecomment-693834761)

**Comment by mtreacy002** [comment]
> > Don't forget there're many ways you could contribute to the project, as mentioned on zulip post here.
> 
> True. Actually I am more inclined towards resolving all the React issues, so I mentioned that apart from that I'll surely contribute wherever I can.
> 
> > Perhaps start a discussion about the design on the #bridge-in-tech stream on the Navbar improvement you mentioned before? 😁
> 
> I guess you confused me with `Prathmesh Khandelwal` he started Navbar discussion [here](https://anitab-org.zulipchat.com/#narrow/stream/237630-bridge-in-tech/topic/New.20Issues/near/210322643)
> Or are you talking about restructuring the project components which I mentioned [here](https://github.com/anitab-org/bridge-in-tech-web/issues/96#issuecomment-693834761)

aaa... yes, sorry, got mixed up 😊.

Anyway, any kind of participation is welcomed 😉. There are few other available issues [here](https://github.com/anitab-org/bridge-in-tech-web/issues) if you're interested. Just ping me if you find one so I can assign you to it 😁

**Comment by HeroicHitesh** [comment]
> There are few other available issues here if you're interested. 

I have no idea about testing in React but would love to learn something about it, so can you redirect me to some useful resources for me to get started with contributing to #33 

> Just ping me if you find one so I can assign you to it 😁

😅Sorry for messaging here, in Zulip PMs are disabled, so I wasn't able to ping you there.

**Comment by mtreacy002** [comment]
@HeroicHitesh , on our AnitaB.Org Open Source zulip, the PM is disabled. However, you don't need to use PM at all to ping me. As part of Open Source value, we encourage open communication. So you can ping me (or anyone you want to reach out) on the channel/topic you want to have discussion on. 

The benefit of pinging using zulip is it's much faster to reach out to member you ping instead of using github. Because zulip app also available on mobile phone and its notification works just like whatsapp or other Mobile Messaging application. 
Posting a comment on github normally will get noticed only when the person check their email, providing you use @<github username> (like @mtreacy002) to reach them on the post. If you don't put this tag+username, your post might get unnoticed until that person check the PR. 
You are lucky that I have the email notification to this repo is set as `on` so I get emailed on your message [above](https://github.com/anitab-org/bridge-in-tech-web/pull/100#issuecomment-694859990) eventhough you didn't put `@mtreacy002` on the post 😁. But I normally check my zulip first before my email.

**Pull request merged**